pixel.py

```
@@ -0,0 +1,55 @@
+                : max(h, rows), : max(w, col), :3
+            ]
+            imgs.append(pix)
+        for i in range(0, rows + (rows & 1), 2):
+            for img in imgs:
+                for j in range(col):
+                    first, *rest = img[i: i + 2, j]
+                    [b1, g1, r1] = first
+                    if rest:
+                        first = rest[0]
+                    else:
+                        first = [0, 0, 0]
```

Ruff (0.8.2)

Use ternary operator `first = rest[0] if rest else [0, 0, 0]` instead of `if`-`else`-block

Replace `if`-`else`-block with `first = rest[0] if rest else [0, 0, 0]`

(SIM108)